chia/cmds/beta_funcs.py

```diff
@@ -127,7 +127,7 @@ def prepare_chia_blockchain_log(path: Path) -> None:
 
 def prepare_logs(prepare_path: Path, prepare_callback: Callable[[Path], None]) -> list[Path]:
     result = [path for path in prepare_path.iterdir()] if prepare_path.exists() else []
-    if len(result):
+    if len(result) > 0:
```

Comment:
Ruff didn't do this; it tried to just strip away the `len(`.  I decided to go the opposite direction and make the check more explicit (same below).